From: Marc Glisse <marc.glisse@inria.fr>
Subject: Complete function prototype in acinclude.m4 for C23 compatibility
Bug-Debian: https://bugs.debian.org/1096730

This patch is the sum of the following two upstream commits:

https://gmplib.org/repo/gmp/raw-rev/8e7bb4ae7a18
https://gmplib.org/repo/gmp/raw-rev/d66d66d82dbb

--- a/acinclude.m4
+++ b/acinclude.m4
@@ -609,7 +609,7 @@
 
 #if defined (__GNUC__) && ! defined (__cplusplus)
 typedef unsigned long long t1;typedef t1*t2;
-void g(){}
+void g(int a,t1 const*b,t1 c,t2 d,t1 const*e,int f){}
 void h(){}
 static __inline__ t1 e(t2 rp,t2 up,int n,t1 v0)
 {t1 c,x,r;int i;if(v0){c=1;for(i=1;i<n;i++){x=up[i];r=x+1;rp[i]=r;}}return c;}
